lib/containers/utils.pm

```diff
@@ -179,4 +179,22 @@ sub test_built_img {
     assert_script_run("rm -rf /root/templates");
 }
 
+=head2 can_build_sle_base
+
+C<can_build_sle_base> should be used to identify if sle base image runs against a
+system that it does not support registration and SUSEConnect B<for job running on OSD>.
+In this case the build of the container engine is not going to work.
+
+The call should return false if you try to test sle base container on osd but host is not sle
+true otherwise.
+
+=cut
+sub can_build_sle_base {
+    my $is_osd = get_var('OPENQA_HOSTNAME', '') =~ /suse.de|suse.cz/;
+    return 0 if $is_osd && check_os_release('opensuse', 'PRETTY_NAME');
```

Comment:
`PRETTY_NAME` is not meant to be read by programs. Use `NAME`, `ID` or `ID_LIKE` instead.

Reply:
no problem with this. But this is what we use in other places too. and in fact it was requested to change from `ID_LIKE`. 
This is open for me anyhow

Reply:
follow the changes in this PR @Vogtinator https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11397